### DISCUSSION TITLE
Migrate from dokuwiki

### DISCUSSION THREAD

**Discussion by FredClement91** [requarks/wiki#4601] [Help / Questions]
Hi all,
I would like to know if there is a simple way to migrate a dokuwiki database to wikijs.

Thanks,

Fred

**Comment by ieaster1** [comment]
It looks like there is a script someone has shared to do a dokuwiki migration: https://github.com/thoni56/dokuwiki2wikijs

Wiki.js 3.0 also has it on the roadmap if it isn't something that is urgent.
https://docs.requarks.io/releases/roadmap#page-management
